### PR TITLE
feat: implement createdByUserId for all features

### PR DESCRIPTION
**Pull request by daveleek**
## About the changes

Implements setting values on the created_by_user_id column on the features table in the db

### CODE REVIEW

File: src/lib/features/feature-toggle/feature-toggle-store.ts [hunk 1]
```diff
@@ -467,20 +473,39 @@ export default class FeatureToggleStore implements IFeatureToggleStore {
             stale: data.stale,
             created_at: data.createdAt,
             impression_data: data.impressionData,
+            created_by_user_id: data.createdByUserId,
```
Comment (chriswk) [inline src/lib/features/feature-toggle/feature-toggle-store.ts:476]: Ah, that was a double take, it looked like you were excluding the createdByUserId and then using it. And then I spotted the fact that you changed it from optional to required. Nice.

**Review by gardleopard — COMMENTED** [review]
Looks good to me:)

**Review by chriswk — APPROVED** [review]
LGTM. Make sure this doesn't break anything in unleash-enterprise as well

**Comment by daveleek** [comment]
Thank you @chriswk! Yeah I've run it with enterprise locally and have all tests passing. I'll keep an eye on the build

**Pull request merged**